kubernetes-server-mock/src/test/java/io/fabric8/kubernetes/client/server/mock/KubernetesCrudAttributesExtractorTest.java

```diff
@@ -326,6 +326,21 @@ public void statusHandling() {
     assertEquals(originalUid, result.getMetadata().getUid());
   }
 
+  @Test
+  public void createConflict() {
+    KubernetesServer kubernetesServer = new KubernetesServer(false, true);
+    kubernetesServer.before();
+    KubernetesClient kubernetesClient = kubernetesServer.getClient();
+    Pod pod = new PodBuilder().withNewMetadata()
+            .withName("name")
+            .withNamespace("test") // required until https://github.com/fabric8io/mockwebserver/pull/59
+            .endMetadata()
+            .withStatus(new PodStatusBuilder().withHostIP("x").build())
+            .build();
+    kubernetesClient.pods().create(pod);
+    assertThrows(KubernetesClientException.class, ()-> kubernetesClient.pods().create(pod));
```

Comment:
nit, Please add an assertion for HTTP_CONFLICT code received in exception.
```
    MixedOperation<Pod, PodList, PodResource<Pod>> podOp = kubernetesClient.pods();
    podOp.create(pod);
    KubernetesClientException exception = assertThrows(KubernetesClientException.class, () -> podOp.create(pod));
    Assertions.assertEquals(HTTP_CONFLICT, exception.getCode());
```

Reply:
okay, I've made that update.